src/ceph-disk/ceph_disk/main.py

```diff
@@ -5436,6 +5452,12 @@ def make_activate_all_parser(subparsers):
         are not activated.
         """)),
         help='Activate all tagged OSD partitions')
+    activate_parser.add_argument(
```

Comment:
s/activate_all_parser/activate_parser/

Reply:
done

Reply:
@dachary sorry, should be activate_all_parser

Reply:
but it's still `activate_parser`. did you forget to push the latest change?

Reply:
yes...